### DISCUSSION TITLE
Is it possible to obtain the 16.04.7.3 Security Onion Iso?

### DISCUSSION THREAD

**Discussion by JoeysRealm** [Security-Onion-Solutions/securityonion#8493] [Unsupported Versions] [answered]

**Comment by Acewiza** [comment]


**Reply by JoeysRealm** [reply]
Thank you :)

**Reply by dougburks** [reply]
I'm sure that @Acewiza has good intentions. However, we recommend against downloading ISO images from non-official sources. We also recommend against using old versions of software that are past End Of Life as they are unsupported and contain unpatched vulnerabilities.

**Comment by dougburks** [comment, marked as answer]
Security Onion 16.04 is past End Of Life and we no longer provide downloads or support for it:
https://docs.securityonion.net/en/2.3/eol.html

We recommend using the latest version of Security Onion 2.3:
https://securityonion.net/download

**Reply by Acewiza** [reply]
Oh yeah, I 2nd that advice!  Always a general rule of thumb on the Internet - hard and fast rule for anything in production.
